spec/cypress/app_commands/clean.rb

```diff
@@ -0,0 +1,10 @@
+if defined?(DatabaseCleaner)
+  # cleaning the database using database_cleaner
+  DatabaseCleaner.strategy = :truncation
+  DatabaseCleaner.clean
+else
+  logger.warn "add database_cleaner or update cypress/app_commands/clean.rb"
+  Post.delete_all if defined?(Post)
```

Comment:
@skaidra-bake lets remove that line and just keep the `logger`

Reply:
@denissellu I'm not sure about this. which lines should be kept and which should be deleted?